template/tox.ini.jinja

```diff
@@ -66,4 +66,4 @@ deps =
 skip_install = true
 changedir = requirements
 commands = python ./make_base.py{% if nightly_deps %} --nightly {{nightly_deps}}{% endif %}
-           pip-compile-multi -d .
+           pip-compile-multi -d . --backtracking
```

Comment:
This was needed to fix a resolving error around prompt-toolkit. As I understand, the backtracking resolver will become the default in the future anyway.